### PR TITLE
#5115 icons fix

### LINKED ISSUE #5115: Share and more options in gallery item
its jumping to the second row even when there is room for it
image

image

on mobile the share button should collapse to only icon 
image

### Changes
- buttons not jumping to second row 
- on mobile change "share" to icon
- more options icon should be square

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #5115
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-02-25 at 5 32 34 PM

### CODE REVIEW

File: styles/components/_gallery-item.scss [hunk 1]
```diff
@@ -20,6 +20,10 @@
       border: 1px solid theme('border-color'); 
     }
   }
+
+  .icon-action {
```
Comment (vikiival) [inline styles/components/_gallery-item.scss:24]: Do not use px

File: components/gallery/GalleryItem.vue [hunk 2]
```diff
@@ -176,6 +176,10 @@ useNuxt2Meta({
   font-size: 2.4375em;
 }
 
+.name-container {
+  max-width: 75%;
+}
+
```
Comment (vikiival) [inline components/gallery/GalleryItem.vue:182]: Use linter

**Review by vikiival — CHANGES_REQUESTED** [review]

File: components/gallery/GalleryItem.vue (hunk 2 above)
Comment (prachi00) [reply components/gallery/GalleryItem.vue:182]: prettier not picking anything wrong though

Comment (roiLeo) [reply components/gallery/GalleryItem.vue:182]: Hey, what does `name-container` fix? I've seen that is working well without

Comment (prachi00) [reply components/gallery/GalleryItem.vue:182]: when its a long name, buttons starts to show up one below the other, max width fixes that

Comment (roiLeo) [reply components/gallery/GalleryItem.vue:182]: oki

**Review by roiLeo — APPROVED** [review]

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 15 usd

**Comment by prachi00** [comment]
@vikiival can we retry payment on this?

**Comment by vikiival** [comment]
It was payed see comment above

**Comment by prachi00** [comment]
> It was payed see comment above

It failed though

**Comment by vikiival** [comment]
> > It was payed see comment above
> 
> 
> 
> It failed though 

Yup, sorry

It will be resolved by #5122

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 5 USD/DOT ~ 4 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x8fefa41c1f733e0c492edd9d55998f08e57ad2594e340a266559a130c8335fb6](https://polkadot.subscan.io/extrinsic/0x8fefa41c1f733e0c492edd9d55998f08e57ad2594e340a266559a130c8335fb6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**